README.md

```diff
@@ -3,49 +3,87 @@
 This project is about harvesting from external sources to a CKAN (local or remote) instance.  
 This project uses a custom Python library ([ckan-harvesters](https://pypi.org/project/ckan-harvesters/))in order to read and validate different types of data sources.  
 
-## data.json files
+Install locally
 
-[More info](harvester_ng/datajson/README.md)
+```
+python3 -m venv /path-to-harvester-env
+source /path-to-harvester-env/bin/activate
+pip install -r requirements.txt
+```
 
-## CSW
+## Run with CKAN 
 
-[More info](harvester_ng/csw/README.md)
+First copy `docker-extras/.env.example` to `docker-extras/.env` file.  
```

Comment:
Is the .env necessary? If the docker setup seems to work with all the defaults, so let's not make the .env a required step. It can be an optional step:

> If you need to customize the environment, copy `docker-extras/.env.example` to `docker-extras/.env` and provide any settings you require.

Reply:
> Is the .env necessary? If the docker setup seems to work with all the defaults, so let's not make the .env a required step. It can be an optional step:
> 
> > If you need to customize the environment, copy `docker-extras/.env.example` to `docker-extras/.env` and provide any settings you require.

Removed the .env. Working with defaults